submissions/maksymenkonat/popup/index.html

```diff
@@ -0,0 +1,161 @@
+                      <p class="text">Mail</p>
+                    </a>
+                  </li>
+                  <li class="menu_li">
+                    <a href="#" class="one_section">
+                      <img src="assets/news.png" alt="News" />
+                      <p class="text">News</p>
+                    </a>
+                  </li>
+                  <li class="menu_li">
+                    <a href="#" class="one_section">
+                      <img src="assets/maps.png" alt="Map" />
+                      <p class="text">Map</p>
+                    </a>
+                  </li>
+                  <li class="menu_li">
+                    <a href="#" class="one_section">
+                      <img src="assets/files.png" alt="Files" />
+                      <p class="text">Files</p>
+                    </a>
+                  </li>
+                  <li class="menu_li">
+                    <a href="#" class="one_section">
+                      <img src="assets/contacts.png" alt="Contacts" />
+                      <p class="text">Contacts</p>
+                    </a>
+                  </li>
+                  <li class="menu_li">
+                    <a href="#" class="one_section">
+                      <img src="assets/translation.png" alt="Translate" />
+                      <p class="text">Translate</p>
+                    </a>
+                  </li>
+                  <li class="menu_li">
+                    <a href="#" class="one_section">
+                      <img src="assets/videos.png" alt="Videos" />
+                      <p class="text">Videos</p>
+                    </a>
+                  </li>
+                  <li class="menu_li">
+                    <a href="#" class="one_section">
+                      <img src="assets/search.png" alt="Search" />
+                      <p class="text">Search</p>
+                    </a>
+                  </li>
+                  <li class="menu_li">
+                    <a href="#" class="one_section">
+                      <img src="assets/photos.png" alt="Photo" />
+                      <p class="text">Photo</p>
+                    </a>
+                  </li>
+                  <li class="menu_li">
+                    <a href="#" class="one_section">
+                      <img src="assets/apps.png" alt="Apps" />
+                      <p class="text">Apps</p>
+                    </a>
+                  </li>
+                  <li class="menu_li">
+                    <a href="#" class="one_section">
+                      <img src="assets/documents.png" alt="Document" />
+                      <p class="text">Document</p>
+                    </a>
+                  </li>
+                </ul>
+                <input type="checkbox" id="more_checkbox" />
+                <label for="more_checkbox" class="menu_checkbox">
+                  <span class="button_more">More</span>
+                </label>
```

Comment:
This button is not accessible from the keyboard, please fix it. Also, hide this button when the second part of the popup is open. [Watch video](https://share.vidyard.com/watch/AjhUWq8C2dd3PzNAy2eynq?)